test/TestCliRolePaths.py

```diff
@@ -1,13 +1,16 @@
 import unittest
 import subprocess
 import os
+import sys
 
 
 class TestCliRolePaths(unittest.TestCase):
     def setUp(self):
         self.local_test_dir = os.path.dirname(os.path.realpath(__file__))
 
-    def run_ansible_lint(self, cwd, bin, role_path, env=None):
+    def run_ansible_lint(self, cwd, role_path, bin=None, env=None):
+        if not bin:
+            bin = sys.executable + " -m ansiblelint"
```

Comment:
Nitpick. There's a trick you may employ to make the string interpolations look closer to f-strings:
```suggestion
            bin = '{sys.executable} -m ansiblelint'.format_map(locals())
```